code/modules/mob/living/living.dm

```diff
@@ -288,20 +288,34 @@
 	visible_message("<b>[src]</b> points to [pointed_object]")
 	return TRUE
 
+/mob/living/proc/can_succumb()
+	return health < HEALTH_THRESHOLD_CRIT && stat == UNCONSCIOUS
+
+
 /mob/living/verb/succumb()
-	set hidden = 1
-	if(InCritical())
-		create_attack_log("[src] has ["succumbed to death"] with [round(health, 0.1)] points of health!")
-		create_log(MISC_LOG, "has succumbed to death with [round(health, 0.1)] points of health")
-		adjustOxyLoss(health - HEALTH_THRESHOLD_DEAD)
-		// super check for weird mobs, including ones that adjust hp
-		// we don't want to go overboard and gib them, though
-		for(var/i = 1 to 5)
-			if(health < HEALTH_THRESHOLD_DEAD)
-				break
-			take_overall_damage(max(5, health - HEALTH_THRESHOLD_DEAD), 0)
-		death()
-		to_chat(src, "<span class='notice'>You have given up life and succumbed to death.</span>")
+	set hidden = TRUE
+	if(!can_succumb())
```

Comment:
I think this should be simplified, I can't think of anywhere else can_succumb should be used

Reply:
ough I kinda meant to get rid of that